Azure/sonic-buildimage
# [udev] Add customize udev rule for console switch(for e1031)

**- Why I did it**
This is for customers customize their own device name for ttyUSB devices

**- How I did it**
Design an udev rule for an assigned console switch; And rename all the ports as assigned prefix.
Here is an console switch called C0 and rename 40 console ports as C0-1 to C0-40.

**- How to verify it**
plugin the C0 use command:
ls /dev/C0* this can check whether we rename success;

**- Which release branch to backport (provide reason below if selected)**
201911

**- Description for the changelog**
This design including following files to change:

*.rules this is for the design of udev rules
popmsg.sh this is for pop msg when the device been plugged out. It shows reminder that the device plug out
udev_prefix.sh this file is for record the name prefix that customers want to use
platform-modules-haliburton.install this is config file to copy files *.rules to the /etc/rules.d so that this rule can be loaded when system reboot
